test/functional-portable/cli/noncloud/cli_test.go

```diff
@@ -68,7 +68,9 @@ func verifyRecipeCLI(ctx context.Context, t *testing.T, test rp.RPTest) {
 	version := strings.TrimPrefix(testutil.GetBicepRecipeVersion(), "version=")
 	resourceType := "Applications.Datastores/redisCaches"
 	file := "../../../testrecipes/test-bicep-recipes/corerp-redis-recipe.bicep"
-	target := fmt.Sprintf("br:ghcr.io/radius-project/dev/test-bicep-recipes/redis-recipe:%s", generateUniqueTag())
+
+	// The target is a local registry, so we can use localhost.
+	target := fmt.Sprintf("br:localhost:5000/dev/test-bicep-recipes/redis-recipe:%s", generateUniqueTag())
```

Comment:
Should we pass this in via an env-var since this requires additional setup? Then we could skip if the env-var is not set.

It's best if the functional tests "just work" without additional steps like a local registry.